### ISSUE TITLE
Replace and use XPlaneProfiler

### ISSUE THREAD

**Issue by tngreene** [der-On/XPlane2Blender#318]
Related to #74, it would be useful, I think, to log how long exporting takes, and perhaps, even, in debug builds, log exactly how long each step of the process takes. At some point we must take into account the export time of the exporter as a feature.

Right now we have no data for how long it takes to run the exporter on a variety of projects, which means we have no baseline. Having a good profiler and using it would be a good step to have for the long term development of this program.

We don't even time how long the test suite takes to run.

**Comment by arb65912** [comment]
I can make a test and export all visible layers, it will take X number of minutes but even if that data is available is there any way to make export faster? 
If not, no point to test in my opinion.

**Comment by tngreene** [comment]
Yes, there are ways to export faster. One big one is our vertex optimizer is very slow, so turning that off is a help - but, we don't know by how much. Another way to make export faster is to optimize the code. If it turns out one dumb algorithm is slowing everything down, we may be able to replace it! But we don't know what to replace if we don't profile, and profiling across time is better than profiling the day of starting an optimization project.

Though I suppose source control can help us profile across time instead of asking for reports from the community... that'll probably be the best.

**Comment by tngreene** [comment]
This project has a hole in it's testing suite, I think. All of our test files are little cubes and rectangles. The only complete plane I have to test is the [BD5J Microjet](https://forums.x-plane.org/index.php?/files/file/27269-bd-5j-microjet/).

It is GNU licensed, so, it might be available to include a version of it in the test suite itself as a real life test. I wish I had more free open-source planes.
